back/sssscs/src/main/java/com/ib/user/UserController.java

```diff
@@ -45,9 +50,12 @@ public ResponseEntity<String> login(@Valid @RequestBody UserLoginDto dto) {
 		} catch (BadCredentialsException ex) {
 			throw new ResponseStatusException(HttpStatus.BAD_REQUEST, "Wrong email or password!");
 		}
+		SecurityContext sc = SecurityContextHolder.getContext();
+		sc.setAuthentication(auth);
```

Comment:
Unnecessary

Reply:
Fixed in [d1743de](https://github.com/magley/sssscs/pull/14/commits/d1743de81b68f2e86d68c45a6f8020838cadc3ee).